config/settings/base.py

```diff
@@ -226,3 +227,8 @@
 )
 
 REDIS_URL = os.environ.get("REDIS_URL", "redis://localhost:6379/0")
+
+# Turn Alerts
+
+TURN_ALERTS_JOURNEY_URL = env.str("TURN_ALERTS_JOURNEY_URL", None)
```

Comment:
We can maybe use the details that we configure per org: https://github.com/praekeltfoundation/rp-sidekick/blob/develop/sidekick/models.py#L21-L22

I think check if they are still being used elsewhere then we take it from there.

The idea with sidekick was always to allow multi-tenancy, for MC both QA and prod run in one instance with different orgs configured. This means you need the org id in the url like the interceptor.